packages/cli/src/methods.ts

```diff
@@ -0,0 +1,126 @@
+import type { MermaidChart } from '@mermaidchart/sdk';
+
+import { CommanderError } from '@commander-js/extra-typings';
+import { extractFrontMatter, injectFrontMatter, removeFrontMatterKeys } from './frontmatter.js';
+
+/**
+ * Cached data to use when pulling/pushing/linking multiple files at once.
+ */
+interface Cache {
+  /**
+   * If set, the user has said to use the projectId to create all documents
+   * in.
+   */
+  selectedProjectId?: string;
```

Comment:
nit: Consistency

```suggestion
  selectedProjectID?: string;
```

Reply:
:+1:

I'll look into fixing all of these before we make the first release, but after #22 is merged, to avoid causing merge conflicts.